pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "kai"
+version = "0.0.1"
+dependencies = [
```

Comment:
Is this replacing requirements.in?

If so, let's also capture how to update a new dependency for contributors in
https://github.com/konveyor/kai/blob/main/CONTRIBUTING.md#modifying-a-python-dependency

Reply:
Yes. Documentation is up next